txt2tags.py

```diff
@@ -561,13 +561,13 @@ def getTags(config):
             "tableTitleCellClose": "</th>",
             "tableTitleCellOpen": "<th~s~>",
             "title1Close": "</section>",
-            "title1Open": "<section~A~>\n<h1>\a</h1>",
+            "title1Open": "<section~A~>\n<h2>\a</h2>", #www3 validator: h1 only for top level
```

Comment:
I believe this is a warning and not an error in the w3 validator. I think it's nice to map the titles to h1-h5, so let's ignore the warning and please undo this change.

Reply:
The warning is:
---------------------------------------------------------
Warning: Consider using the h1 element as a top-level heading only (all h1 elements are treated as top-level headings by many screen readers and other tools).
From line 330, column 1; to line 330, column 4
<section>↩<h1>Specia
---------------------------------------------------------
in the w3 spec I did not found an indication for this warning, so I think it is ok to start with h1